go.mod

```diff
@@ -1,15 +1,14 @@
 module github.com/ipfs/dnslink-dnsimple
 
-go 1.20
+go 1.21
```

Comment:
@ns4plabs I think our CI requires go 1.20 here, so the 1.21 becomes the "next" target.

Reply:
fixed in https://github.com/ipfs-shipyard/dnslink-dnsimple/pull/21/commits/9cf491f5c004800e9e85ec8e951113714a47d344